_sass/blocks/revenues/_revenues_subpage_diving_graphics.scss

```diff
@@ -69,11 +70,12 @@
   &.offshore-oil{
 
     background: $offshore-oil-primary-color;
-    
+
     .revenues_subpage-dyk_heading {
       background-color: $offshore-oil-secondary-color;
     }
-    .revenues_subpage-steps_heading {
+    .revenues_subpage-steps_heading,
```

Comment:
Selector `revenues_subpage-steps_heading` should be written in lowercase with hyphens<br>Selector `revenues_subpage-steps_heading_more` should be written in lowercase with hyphens